docs/DeveloperGuide.md

```diff
@@ -294,37 +294,43 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (un
 |Priority|As a …​    |I want to …​                                                  |So that                                                                                         |
 |--------|--------------|-----------------------------------------------------------------|------------------------------------------------------------------------------------------------|
 |* * *   |HR Executive  |allocate drivers to passengers to be picked up                   |I can arrange carpooling trips for my colleagues                                                |
-|* * *   |HR Executive  |search for specific type of passengers                           |I can see if any passengers fulfil a criteria and view their carpool details                    |
+|* * *   |HR Executive  |search for specific type of passengers                            |I can see if any passengers fulfil a criteria and view their carpool details                    |
 |* * *   |HR Executive  |list all passengers                                              |I can see all the passengers available                                                          |
-|* * *   |HR Executive  |create a profile                                                 |I can easily manage and track drivers and passengers                                            |
-|* * *   |HR Executive  |delete employee profile                                          |passenger's data will not be stored when they are no longer looking to carpool                  |
-|* *     |HR Executive  |filter passengers' destination and pickup point based on location|drivers are not heavily inconvenienced to pick up passengers                                    |
-|* *     |HR Executive  |match only with female drivers                                   |so that female colleagues looking to carpool only with female drivers can be easily accomodated |
```

Comment:
how about if we were to say that the passengers are filtered by femaleDrivers only

Reply:
Yeap ok that would make sense. Ill rephrase the user story.